Tests/Functional/app/Basic/bundles.php

```diff
@@ -10,13 +10,11 @@
  */
 
 use FOS\ElasticaBundle\FOSElasticaBundle;
-use Knp\Bundle\PaginatorBundle\KnpPaginatorBundle;
 use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
 use Symfony\Bundle\TwigBundle\TwigBundle;
 
 return [
     new FrameworkBundle(),
     new FOSElasticaBundle(),
-    new KnpPaginatorBundle(),
```

Comment:
there is no functional test that uses this (there is PaginateElasticaQuerySubscriberTest which is a unit test and only needs knplabs/knp-components)